wremnants/plot_tools.py

```diff
@@ -33,7 +33,11 @@ def figure(href, xlabel, ylabel, ylim=None, xlim=None,
     raw_width = (hax.size/float(bin_density)) * (xlim_range / original_xrange)
     width = math.ceil(raw_width)
 
-    fig = plt.figure(figsize=(width_scale*8*width,8))
+    if automatic_scale:
```

Comment:
I don't quite understand how automatic_scale is different than width=1

Reply:
Setting automatic_scale to false sets width=1. I re structured the part of the code a little bit